inloop/solutions/views.py

```diff
@@ -104,10 +106,20 @@ def get(self, request, **kwargs):
             junit.xml_to_dict(xml) for xml in xml_reports
         ]
 
+        upfiles = []
+        for solutionfile in solution.solutionfile_set.all():
+            with solutionfile.absolute_path as f:
+                upfiles.append({
+                    "code": f.open(encoding="utf-8").read(),
+                    "title": solutionfile.name,
+                    "size": os.path.getsize(str(f))
```

Comment:
Please use `solutionfile.absolute_path.stat().st_size` (see https://docs.python.org/3/library/pathlib.html#pathlib.Path.stat) instead of the old functions from the `os.path` package.